packages/flutter_rx_bloc/CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
+## [4.1.0] - May 24, 2022
```

Comment:
Since we introduce a breaking change, we must bump the version of the package to 5.0.0